healthcheck/healthcheck_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package healthcheck
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy"
+)
+
+func newClient(mc uint64) *proxy.Client {
+	return &proxy.Client{
+		MaxConnections: mc,
+	}
+}
+
+func TestLiveness(t *testing.T) {
+	proxyClient := newClient(0)
+	InitHealthCheck(proxyClient)
```

Comment:
Here's a good example of where we'll want to be able to call "Shutdown" or "Close" on the healthcheck. Each of these starts a separate server without shutting them down.